accounts/views.py

```diff
@@ -34,3 +36,29 @@ def get_queryset(self):
         return Account.objects.all()
 
 
+class AccountCreateView(generics.CreateAPIView):
```

Comment:
Again, use ModelViewSet for list and CRUD

Reply:
it's below sir after testing I will remove generics